allan_variance.py

```diff
@@ -67,14 +73,17 @@ def allan_variance(x, dt=1, min_cluster_size=1, min_cluster_count='auto',
     log_max = np.log2((n - min_cluster_count) // 2)
 
     cluster_sizes = np.logspace(log_min, log_max, n_clusters, base=2)
-    cluster_sizes = np.unique(np.round(cluster_sizes)).astype(int)
+    cluster_sizes = np.unique(np.round(cluster_sizes)).astype(np.int64)
```

Comment:
Yes, I forgot about that, sorry, I guess int64 is good if we want to work with *very* large arrays. But when computing AVAR I believe we need to do something to avoid overflow, like:

```Python
avar[i] = np.mean(c**2, axis=0) / k / k
```
(Looks weird, but I think it is the right way).